### PR TITLE
terraform-modules and blueprints orgs

### PR DESCRIPTION
**Pull request by anton-sidelnikov**

### CODE REVIEW

File: orgs/opentelekomcloud-blueprints/people/members.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+users: []
```
Comment (outcatcher) [inline orgs/opentelekomcloud-blueprints/people/members.yml:2]: Why no users?

File: orgs/terraform-opentelekomcloud-modules/people/members.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+users:
+  - name: "Rodion Gyrbu"
+    login: "lego963"
+    visibility: Public
```
Comment (outcatcher) [inline orgs/terraform-opentelekomcloud-modules/people/members.yml:5]: Why there is only one member?

**Review by outcatcher — CHANGES_REQUESTED** [review]

File: orgs/opentelekomcloud-blueprints/people/members.yml (hunk 1 above)
Comment (anton-sidelnikov) [reply orgs/opentelekomcloud-blueprints/people/members.yml:2]: no users in this organization, idk why

File: orgs/terraform-opentelekomcloud-modules/people/members.yml (hunk 2 above)
Comment (anton-sidelnikov) [reply orgs/terraform-opentelekomcloud-modules/people/members.yml:5]: you want to be added?

Comment (anton-sidelnikov) [reply orgs/terraform-opentelekomcloud-modules/people/members.yml:5]: Currently only Rodion included

Comment (outcatcher) [reply orgs/terraform-opentelekomcloud-modules/people/members.yml:5]: Currently there are four members:
image

File: orgs/opentelekomcloud-blueprints/people/members.yml (hunk 1 above)
Comment (outcatcher) [reply orgs/opentelekomcloud-blueprints/people/members.yml:2]: Maybe you don't have enough permissions and they are hidden?

File: orgs/terraform-opentelekomcloud-modules/people/members.yml (hunk 2 above)
Comment (anton-sidelnikov) [reply orgs/terraform-opentelekomcloud-modules/people/members.yml:5]: okay

**Review by outcatcher — APPROVED** [review]

**Pull request merged**